src/Microsoft.TestPlatform.PlatformAbstractions/netcore/System/ProcessHelper.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.VisualStudio.TestPlatform.PlatformAbstractions
+{
+    using System.IO;
+    using System.Reflection;
+    using Microsoft.VisualStudio.TestPlatform.PlatformAbstractions.Interfaces;
+
+    public partial class ProcessHelper : IProcessHelper
+    {
+        /// <inheritdoc/>
+        public string GetCurrentProcessLocation()
```

Comment:
Method name won't match defination, Should we rename to EntryPointDirectory?

Reply:
from users perspective in netcore, the process would be the dll he/she is creating.